### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by ralcorta**
This PR aims to add more information and examples on how to use XML as a parameter instead of just a URL or PATH to create the SOAP client. This method is not explained anywhere in the README, and I believe it is crucial to include it since it is a feature of the package that currently lacks visibility.

The relevant code is found at this line: https://github.com/vpulim/node-soap/blob/master/src/wsdl/index.ts#L1406

### CODE REVIEW

**Comment by w666** [comment]
Hi @ralcorta,

Thanks for submitting this improvement!

**Review by w666 — APPROVED** [review]

**Comment by ralcorta** [comment]
Hi @w666 ! Thanks for the approval, should I tag someone else to merge it?

**Comment by w666** [comment]
> Hi @w666 ! Thanks for the approval, should I tag someone else to merge it?

Nah, I am a maintainer at the moment. I will merge it when get enough changes for the next release.
Likely it will be merged at the same time with https://github.com/vpulim/node-soap/pull/1242/.

**Pull request merged**
